assets/js/translator.js

```diff
@@ -743,11 +743,32 @@ function translateDoc() {
                         downloadBrowserWarn();
                         $('div#fileUploadProgress').parent().fadeOut('fast');
                         $('div#fileLoading').fadeOut('fast', function () {
-                            var URL = window.URL || window.webkitURL;
-                            $('a#fileDownload')
-                                .attr('href', URL.createObjectURL(xhr.response))
-                                .attr('download', fileName)
-                                .fadeIn('fast');
+                            if(typeof window.navigator.msSaveBlob !== 'undefined') {
+                                $('a#fileDownload')
```

Comment:
Could you add a one-liner comment to each of these conditional blocks stating what browser the support is for?